### PR TITLE
Author Credit

### PR DESCRIPTION
**Pull request by BlckHawker**
Added author credit to read page
- Clicking on the profile picture will show more info (current just when the panel set was created)
- Refactored `InfoBox` and  `InfoBtn` to handle its visibility

### CODE REVIEW

File: src/components/ReadPage.tsx
Comment (4rt3ry) [inline src/components/ReadPage.tsx:1]: Created date is invalid
image

File: src/components/Signature.tsx
Comment (4rt3ry) [inline src/components/Signature.tsx:1]: Add CSS on hover
```css
cursor: pointer;
```

**Review by 4rt3ry — CHANGES_REQUESTED** [review]

File: src/components/ReadPage.tsx
Comment (BlckHawker) [reply src/components/ReadPage.tsx:1]: I'm not getting that error. Make sure you backend is up to date with main. If the error still occurs, would you mind printing out what `author?.created_at` is for you on line 186 of the read page?

Comment (4rt3ry) [reply src/components/ReadPage.tsx:1]: yep, it works

**Pull request merged**